codebuild_specs/build_tests_standalone-yarn-3.yml

```diff
@@ -0,0 +1,11 @@
+version: 0.2
+env:
+  shell: bash
+phases:
+  build:
+    commands:
+      - echo "Build Tests Standalone"
+      - source ./shared-scripts.sh && _buildTestsStandaloneYarn3
```

Comment:
Can you add a comment on how this is testing the issue fixed in this PR?

Reply:
I think the name of the test is self-explanatory because the PR fix the issue for yarn3 and the test just verify if yarn3 works.

Reply:
my confusion stems from the fact that the test is asserting that we can build our own tests using yarn3(not even running them) while the fix seems to be for customers to be able to build their functions using yarn3. So I might be missing something here on how these two are related.

Reply:
Good point. I re-did the test de785772eb2e6da7c2554153e46b0ac43d30a328